app/components/App/App.js

```diff
@@ -18,7 +17,9 @@ class App extends Component {
             <Route exact path='/' component={Play} />
             <Route exact path='/instructions' component={Instructions}/>
             <Route path='/learn' component={Learn} />
-            <p className='copyright-notice'>&#169; 2017 Leta Keane</p>
+            <footer>
+              <p className='copyright-notice'>&#169; 2017 Leta Keane</p>            
+            </footer>
```

Comment:
Oh look! The land of make believe!!!!!!!!! 

No, good idea! This project is so incredibly interesting.